spec/stud/interval_spec.rb

```diff
@@ -1,8 +1,19 @@
 require "stud/interval"
 require "spec_env" # from the top level spec/ directory
 
+module Stud
+  def reset!(target = Thread.current)
```

Comment:
More than just expose it, it should by default cleanup the Thread locals.

Do you remember the old days of assigning the `current_user` from a rails app  in a thread local? ;)